test/Bin/GameTest/GameTests_1000.cpp

```diff
@@ -466,6 +466,7 @@ GAME_TEST(Issues, Issue1383) {
     character.pActiveSkills[CHARACTER_SKILL_MERCHANT] = CombinedSkillValue();
     int noobPrice = PriceCalculator::itemBuyingPriceForPlayer(&character, item.GetValue(), 10.0f);
     EXPECT_EQ(noobPrice, 75000);
+    uCurrentlyLoadedLevelType = LevelType::LEVEL_NULL;
```

Comment:
Why is this needed? Let's either add a comment, or a TODO

Reply:
uCurrentlyLoadedLevelType was getting carried into future tests. It was trying to draw an indoor level before anything was loaded.